ansible/roles/vm_set/files/mux_simulator.py

```diff
@@ -809,6 +809,22 @@ def reset_flow_handler(vm_set):
     return g_muxes.reset_flows()
 
 
+@app.route('/mux/<vm_set>/output', methods=['POST'])
+def output_flow_handler(vm_set):
+    _validate_vm_set(vm_set)
+    data = _validate_out_sides(request)
+    app.logger.info('===== {} POST {} with {} ====='.format(request.remote_addr, request.url, json.dumps(data)))
+    return g_muxes.update_flows('output', data['out_sides'])
+
+
+@app.route('/mux/<vm_set>/drop', methods=['POST'])
+def drop_flow_handler(vm_set):
+    _validate_vm_set(vm_set)
+    data = _validate_out_sides(request)
```

Comment:
Can you add docstring to explain expected data schema in POST request?
Also can you update the mux simulator README file to document these two new APIs?

Reply:
Accepted. Thanks!